mkdocs.yml

```diff
@@ -114,8 +114,8 @@ extra_javascript:
 
 
 nav:
-  - Welcome: https://docs.kedro.org/en/1.0.0rc1/
-  - Kedro: https://docs.kedro.org/en/1.0.0rc1/getting-started/course/
+  - Welcome: https://docs.kedro.org/en/stable/
+  - Kedro: https://docs.kedro.org/en/stable/getting-started/course/
```

Comment:
Until 1.0 is released, some of these links will be broken, but also we don't want to hardcode a specific version here. Let's just have this in mind for the next couple of weeks